### PR TITLE
ENH: Update CI build for ITK 5.2 RC 1

### PR DESCRIPTION
**Pull request by thewtex**
Custom tag is required for issues in the C++ builds.

Python packages have 4D support to help address #70

### CODE REVIEW

**Review by dzenanz — APPROVED** [review]

**Review by ViktorvdValk — APPROVED** [review]
Looks good!
4D registration is still not working, but Elastix is the issue this time, so I'll see if that can be fixed as well.

File: .github/workflows/build-test-package.yml [hunk 1]
```diff
@@ -13,17 +13,17 @@ jobs:
           - os: ubuntu-18.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "v5.1.2"
+            itk-git-tag: "abd38d5a0040b9a8fbb0ad3127089dbb72a93342"
```
Comment (N-Dekker) [inline .github/workflows/build-test-package.yml:16]: Should this itk-git-tag also be "v5.2rc01", rather than "abd38..."?

Comment (dzenanz) [reply .github/workflows/build-test-package.yml:16]: No, it should be abd38d5. There were some fixes added after RC1 was tagged.

Comment (N-Dekker) [reply .github/workflows/build-test-package.yml:16]: @dzenanz Thanks, so could you say which fixes were essential for the ITKElastix CI build? Maybe it's time for RC2... 😸

Comment (dzenanz) [reply .github/workflows/build-test-package.yml:16]: Yes, RC2 is planned 😄

Meanwhile we should merge this.

**Review by N-Dekker — APPROVED** [review]

**Pull request merged**